pkg/config/config_template.yaml

```diff
@@ -1480,6 +1480,13 @@ api_key:
   ## may not be marked by the Agent as top-level spans.
   # peer_service_aggregation: false
 
+  ## @param peer_tags - list of strings - optional
+  ## @env DD_APM_PEER_TAGS - list of strings - optional
+  ## Enables additional tags related to peer.service. Will not be used unless peer_service_aggregation is also set.
+  ## NOTE: This option is recommended to get the most granular tagging alongside `peer.service`, but it will
+  ## also increase the computational overhead of aggregation. This list can be omitted if that cost is too high for your agent.
+  # peer_tags: false
```

Comment:
> peer_tags: false

This shouldn't be a boolean here, right? Isn't it a list of strings?

Reply:
Right! Will fix.